bundle

```diff
@@ -36,6 +36,8 @@ class CharmstoreQueryError(Exception):
 def get_args():
     ''' Parse cli arguments. '''
     parser = argparse.ArgumentParser()
+    parser.add_argument('-n', '--name', default=None,
+                        help='name given to the created bundle.')
```

Comment:
a valid `bundle.yaml` must have a name.  If you don't provide one, one will be generated based on the path name